### DISCUSSION TITLE
code annotation in title

### DISCUSSION THREAD

**Discussion by soxft** [vercel/next.js#61281] [Help] [answered]
### Summary

After I deploy my code in vercel, I found that there seemed to be comments in the title on the first screen of my website, and the comments were removed after the JavaScript was loaded. This seems to affect SEO and other issues. Is there any way not to add comments?

<img width="662" alt="image" src="https://github.com/vercel/next.js/assets/42080379/05eb59d0-bafa-408e-bbab-33f61fa66d28">

source code: https://github.com/soxft/home_page_nextjs

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This has to do with how React Server renders strings with interpolations. You have this:

```jsx
      <Head>
        <title>{t('title')} - {t('subtitle')}</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />
      </Head>
```

Change it to:

```jsx
const title = `${t('title')} - ${t('subtitle')}`

// later on...

      <Head>
        <title>{title}</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />
      </Head>

```

Another way to fix it would be:

```jsx
      <Head>
        <title>{`${t('title')} - ${t('subtitle')}`}</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />
      </Head>

```

I have written extensively about this in this answer: https://github.com/vercel/next.js/discussions/38256#discussioncomment-3070196
